src/test/components/ng-http-loader.component.spec.ts

```diff
@@ -347,12 +347,12 @@ describe('NgHttpLoaderComponent', () => {
 
             // the HTTP request is pending for 1 second now
             tick(1000);
-            component.isSpinnerVisible$.subscribe(v => expect(v).toBeFalsy());
+            component.isSpinnerVisibleSubject.subscribe(v => expect(v).toBeFalsy());
```

Comment:
You're not following the pattern `.subscribe` -> `.unsubscribe()` you've been doing on other assertions. Any particular reason?

Reply:
Fixing now !